llama_index/vector_stores/elasticsearch.py

```diff
@@ -168,8 +186,8 @@ def __init__(
         self.batch_size = batch_size
         self.distance_strategy = distance_strategy
 
-        if es_client is not None:
-            self._client = es_client
+        if async_es_client is not None:
```

Comment:
suggestion: continue to call the parameter es_client but we add in the documentation this requires the async version of Elasticsearch.